sales_team_security/migrations/12.0.3.0.0/post-migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """At installation time, need to followers sync"""
+    partners = env['res.partner'].search([
```

Comment:
This should be a post_init_hook, but also being called from this migration script.